lago/validation.py

```diff
@@ -0,0 +1,21 @@
+import logging
+import imp
+
+LOGGER = logging.getLogger(__name__)
+
+
+def check_import(module_name):
```

Comment:
thanks - the idea was to avoid actually importing the module you want to check, I truly hope there are no side-affects here.
btw - don't think this is python 3 compatible.